alert.go

```diff
@@ -563,6 +588,18 @@ func (a *AlertNode) determineLevel(now time.Time, fields models.Fields, tags map
 			continue
 		} else if pass {
 			return AlertLevel(l)
+		} else {
+			lrse := a.levelResets[l]
```

Comment:
I get you. My idea was that when a normal expression returns false, a data point should be evaluated for lower levels only if a reset expression doesn't exists, or if it exists and return true.
I'll try to come up with the tests. Feel free to point out where I get it wrong. : )